packages/npm/@amazeelabs/bridge-waku/src/client.tsx

```diff
@@ -0,0 +1,33 @@
+'use client';
+import type { LocationProviderType, useLocationType } from '@amazeelabs/bridge';
+import { useEffect, useState } from 'react';
+import { useRouter_UNSTABLE } from 'waku';
+
+export { createLinkComponent } from './link.js';
+
+export const createUseLocationHook: (
+  useRouter: typeof useRouter_UNSTABLE,
+) => useLocationType = (useRouter) => () => {
+  const router = useRouter();
+
+  // TODO: Use router.hash when https://github.com/dai-shi/waku/pull/746 is merged.
+  const [hash, setHash] = useState('');
+  useEffect(() => {
+    setHash(window.location.hash);
+  }, [router]);
```

Comment:
Same question as here https://github.com/dai-shi/waku/pull/746/files#r1645256634 🙂

Reply:
There is a very small chance that `window.location.hash` changes without the router changing, and I think that case would rather be a bug somewhere else 🤔 k

But I think i would rather fix the Waku-PR and remove this entirely when we can update.